src/test/utils.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/*
+ * Prepare for mongodb connection to test.
+ */
+
+let mongoose = require('mongoose');
+
+beforeEach(function (done) {
+  function clearDB() {
+    /*
+     * This part is hided because it always cleanDB even when dev-mode,
```

Comment:
I think we can bring this back if we use test-specific db (e.g. yoda-test) for all the tests.